auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java

```diff
@@ -46,14 +50,8 @@ public void onClick(View v){
 //                new TwilioAPI().execute(application);
 //                Toast.makeText(MainLobbyActivity.this, "Parientes notificados", Toast.LENGTH_SHORT).show();
 
-                String sms = "This is going to be a really long message and it will take some time to get there,  This is going to be a really long message and it will take some time to get there";
-
-                SMS message = new SMS(getApplicationContext());
-                message.sendSMS("8317760235",  sms);
-
-
-
-
+                String sms = "this is going to be around 160 chars. testing. this is going to be around 160 chars. testing.this is going to be around 160 chars. testing.this is going to be .";
+                message.sendSMS("4159881453",  sms);
```

Comment:
@dariomolina93 curiously, who's phone number is 4159881453 ?  Thanks.

Reply:
@lloydchang It's actually my phone number, I forgot to remove it lol. I'm going to cancel this PR and submit another with cleaner code. Thanks!

Reply:
Thank you @dariomolina93; sounds good :-)